py/ai/h2o/sparkling/ml/algos/H2OAutoML.py

```diff
@@ -16,52 +16,26 @@
 #
 
 from pyspark import keyword_only
-from pyspark.ml.util import JavaMLWritable, JavaMLReadable
-from pyspark.ml.wrapper import JavaEstimator
+from pyspark.sql.dataframe import DataFrame
 
 from ai.h2o.sparkling import Initializer
-from ai.h2o.sparkling.ml.algos.H2OAutoMLExtensions import H2OAutoMLExtensions
-from ai.h2o.sparkling.ml.models import H2OMOJOModel
+from ai.h2o.sparkling.ml.algos.H2OAlgoBase import H2OAlgoBase
 from ai.h2o.sparkling.ml.params import H2OAutoMLParams
-from ai.h2o.sparkling.sparkSpecifics import get_input_kwargs
 
 
-class H2OAutoML(H2OAutoMLParams, H2OAutoMLExtensions, JavaEstimator, JavaMLReadable, JavaMLWritable):
+class H2OAutoML(H2OAutoMLParams, H2OAlgoBase):
 
     @keyword_only
-    def __init__(self, featuresCols=[], labelCol="label", allStringColumnsToCategorical=True, columnsToCategorical=[], splitRatio=1.0, foldCol=None,
```

Comment:
We can actually optimize the code like this so we don't duplicate any information from Java side